eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/index.test.js

```diff
@@ -1,190 +1,204 @@
 import React from "react";
-import { shallow } from "enzyme";
-import { render, fireEvent } from "tests/utils/rtl";
-import {
-  UnwrappedNavigationSidebar as NavigationSidebar,
-  sidebarActionTypes,
-  sidebarReducer,
-  accordionActionTypes,
-  accordionGroupReducer,
-} from "./";
-import { SynchronousPromise } from "synchronous-promise";
-import {
-  buildQuestionnaire,
-  buildSections,
-} from "tests/utils/createMockQuestionnaire";
 
-describe("NavigationSidebar", () => {
```

Comment:
A lot of these tests are already covered by other ones:

-  `AddMenu.test.js` includes tests for adding a page, section, confirmation, calculated summary and when each should be disabled. 
- An introduction page being featured on the questionnaire isn't a behavior dictated by the side nav panel, so that one isn't needed here. 
- The navigation sidebar doesn't have reducers anymore, so these aren't needed.
- Opening and closing the collapsible toggles is covered in `CollapsibleNavigationItem.test.js`

I can add a test for the Open/Close all button, 'though. I think that's the only one that isn't covered elsewhere. I'll check that before hand 'though to be sure.